client/executor/src/wasm_runtime.rs

```diff
@@ -343,6 +356,57 @@ fn decode_version(version: &[u8]) -> Result<RuntimeVersion, WasmError> {
 	}
 }
 
+fn decode_runtime_apis(mut apis: &[u8]) -> Result<Vec<([u8; 8], u32)>, WasmError> {
+	use std::convert::TryInto;
+
+	if apis.len() % 12 != 0 {
```

Comment:
Maybe this `12` could be some constant somewhere? Instead that it is repeated everywhere in the source code? :D

Reply:
Yeah, good point. Perhaps we should also move the deserialization near to serialization.